### PR TITLE
Fix/htmlattrs - escapeText styles also and handles false attrs correctly

### PR DESCRIPTION
**Pull request by fritx**
Hello @brainkim 
This library is awesome and creative, thanks for this !
I'll try to understand it in the next few days.

Here is my first PR on html attrs, thanks.

### CODE REVIEW

File: src/html.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ function printAttrs(props: IntrinsicProps<string>): string {
 				attrs.push(`${escapeText(name)}="${value}"`);
 				break;
 			case typeof value === "boolean":
```
Comment (brainkim) [inline src/html.ts:61]: `case value === true:`

Comment (brainkim) [reply src/html.ts:61]: Then we don’t have to have an if statement in the body. Thanks for catching this.

Comment (fritx) [reply src/html.ts]: You're right 😆  Fixed.
Feel free to squash my commits or just ask me to do it if needed.

**Comment by brainkim** [comment]
Thank you for the fixes! I will make sure to deploy this ASAP. We probably want a more robust test suite to test XSS vulnerabilities but I appreciate the test.

**Comment by brainkim** [comment]
Deployed in v0.1.1

**Pull request merged**